src/utilities/helper_util.ts

```diff
@@ -162,3 +162,14 @@ export function to_uri_params(params: Hash<string | number | boolean>) {
 export function basic_auth(username: string, password: string): string {
   return "Basic " + Buffer.from(`${username}:${password}`).toString("base64");
 }
+
+export function isServerMode(): boolean {
+  const host = process.env.VUE_APP_API_URL;
```

Comment:
I think we can replace this actually being an app URL with either window.location or just using relative paths everywhere.

This variable would just become a boolean.

Reply:
add server mode as a data item in the vuex state

Reply:
likely then that we would add an elment for is_logged_in to the vuex state as well

Reply:
I think what @aaronlippold and I mentioned are two different items.

Reply:
@Bialogs VUE_APP_API_URL is backend API url. Doesn't window.location just give you the URL of the frontend?

Reply:
Yes it would be the location of where the app was served out of. If you're at hiemdall.com/report/1 you can use window.location to get heimdall.com then do API calls against heimdall.com/api/login

Reply:
@Bialogs @aaronlippold made some updates to the code.